JPF/bad_buildings.sql

```diff
@@ -97,7 +131,7 @@ left join
 on
 	c.complaintid = cp.complaintid
 left outer join 
-    hpd_buildings b
+    hpd_buildings_temp b
 on 
    c.buildingid = b.buildingid
 group by 
```

Comment:
Using the full column name for group by here would be more readable, though it's not essential.